### PR TITLE
Add description

### PR DESCRIPTION
**Pull request by dkovari1**
I will continue describing the project in later, but I don't want this branch to just stay in the void.

### CODE REVIEW

**Comment by markkovari** [comment]
@dkovari1 can you please resolve the conflicts? After that It can be merged.

File: features/Datamanagement/AddData.feature [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Feature: Adding data
+
+
+    Scenario: User add excercises to list
+        Given User fills Date, Type and Length fields
+        When User click on "Add to list" button
+        Then Datas will be added on the bottom of the table
```
Comment (markkovari) [inline features/Datamanagement/AddData.feature]: There is no such thing as `Datas`.

**Comment by dkovari1** [comment]
Ok

File: features/Datamanagement/AddData.feature (hunk 1 above)
Comment (dkovari1) [reply features/Datamanagement/AddData.feature]: "Datas" changed to "Data"

**Review by dkovari1 — COMMENTED** [review]
Changes were made. Datas are now Data

**Review by markkovari — APPROVED** [review]

**Pull request merged**